router/nginx.conf

```diff
@@ -82,6 +82,9 @@ http {
       proxy_set_header        Host $host;
       proxy_pass              http://v2;
     }
+    location /what-we-do/our-collections-online {
+      return 301 https://wellcomecollection.org/works;
+    }
```

Comment:
Where has this URL come from?
If it's Drupal - could we set the redirect up in there please?

Reply:
We can do that now if need be.
Let's keep this PR for WI.